kubernetes/samples/scripts/domain-lifecycle/startCluster.sh

```diff
@@ -0,0 +1,120 @@
+# !/bin/sh
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/helper.sh
+
+function usage() {
+
+  cat << EOF
+
+  This script starts a WebLogic cluster in a domain by patching
+  'spec.clusters[<cluster-name>].serverStartPolicy' attribute of the domain
+  resource to 'IF_NEEDED'. This change will cause the operator to initiate
+  startup of cluster's WebLogic server instance pods if the pods are not
+  already running.
```

Comment:
add "and the `spec.replicas` or `spec.cluster.CLUSTERNAME.replicas` is set higher than zero".